blog.css

```diff
@@ -0,0 +1,63 @@
+body {
+  font-family: 'Josefin Slab', serif;
+  font-size: 1.2rem;
+}
+
+/* Header */
+header {
+  background-image: url(images/header-image.jpg);
+}
+
+header img {
+  max-width: 35%;
+  height: auto;
+  /* float: left; */
+}
+
+header p {
+  color: green;
+  font-size: 1.5rem;
+  font-style: italic;
+  font-weight: bold;
+}
+
+.checkout-page-link {
+  color: white;
+  float: right;
```

Comment:
I'd love for you to not use floats. Floats are hard to manage and not the best way. Think of how you could use flexbox, grid or even positioning